index.html

```diff
@@ -20,13 +23,166 @@
         
     </nav>
     <section class="headline">
-        <div class="group48">
+        <nav class="group48">
             <h1 id="title">Charlie Giorgania</h1>
             <p id="text">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.</br></br> you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
             <button id="group19">Start collaboration</button>
-        </div>
-        <img id="img2" src="image_geometry_2.png" alt="">
+        </nav>
+        <img id="img2" src="image_geometry_2.svg" alt="">
     </section>
+    <section class="work">
+        <h2 id="titleh2">Projects</h2>
+        <nav class="groupWork">
+            <div class="subGroup">
+                <img id="Rectangle21IG" src="Rectangle 21.svg" alt="">
+                <div class="groupInfo">
```

Comment:
- [ ] Kindly as suggested in the project requirements, I think it would be better In order to lay out the elements in `Works` section, to use **CSS Grid** and **Flexbox** combined (it is mandatory that you use both). 🙏 

   Use **CSS Grid** in the main container that contains all of the cards and use **Flexbox** to lay out an individual card. 👍 

_Using both **CSS Grid** and **Flexbox** together provides a flexible and powerful layout system that can handle various layout requirements efficiently. It allows you to create complex and responsive designs with ease and maintain a clear separation of concerns between the overall layout structure and the internal positioning of elements within the cards._